experiments/openebs/openebs-pool-network-loss/openebs_pool_network_loss_k8s_job.yml

```diff
@@ -52,11 +52,8 @@ spec:
             value: '100' # in percentage
 
           - name: TOTAL_CHAOS_DURATION
-            value: '120000' # in milliseconds
+            value: '120'
           
```

Comment:
` KAFKA_CONSUMER_TIMEOUT` will be in ms ?

Reply:
I am not sure about it so I haven't changed. cc: @ksatchit